recipes/lm-sensors/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -ex
+
+export CFLAGS="-I${PREFIX}/include $CFLAGS"
+export LDFLAGS="-L${PREFIX}/lib -Wl,-rpath,${PREFIX}/lib -Wl,-rpath-link,${PREFIX}/lib $LDFLAGS"
```

Comment:
```suggestion
```

Reply:
I tested locally and your tests pass with these changes. 

let me know if it is ok.